modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/segregated/2d/2d-segregated-velocity-rz-slip.i

```diff
@@ -186,10 +193,35 @@ pressure_tag = "pressure_grad"
 []
 
 [FunctorMaterials]
-  [darcy]
-    type = ADGenericFunctorMaterial
-    prop_names = 'Darcy_coefficient Forchheimer_coefficient'
-    prop_values = '0.1 0.1'
+  [friction_coefficient_quad_x]
```

Comment:
add some comments in these inputs that you adding this for compatibility

Reply:
done